kleur.d.ts

```diff
@@ -42,4 +42,4 @@ interface Kleur {
 }
 
 declare let kleur: Kleur & { enabled: boolean };
-export = kleur;
+export default kleur;
```

Comment:
This is a regression from #18 The way I understand is is that CommonJS exports have to be exported as `export = FOO`, since `export default FOO` would (in CommonJS terms) be `exports.default = FOO` explicitly.

I might be mistaken, but it's how I've made sense of it

Reply:
From what i know, the `export = FOO` and `export let foo: Bar` is necessary only for code generation, and not for type definition. I will do some additional testing in a CJS only environment to test the claim. Or maybe @marvinhagemeister has another idea or maybe a use case where the `export default` creates the ambiguity since it would affect some of my other own projects as well 😨

Reply:
Okay indeed it does not properly type the CJS required `kleur` with it. I will try to find a better solution for it 🙇

Reply:
Thanks for looking into it :) 

FWIW, here's what I found about the differences: https://github.com/Microsoft/TypeScript/issues/7185

I believe it plays a role during ts-check _and_ building your project, since it always has to know where modules' code is coming from & how to assemble it.

I think on your side, you need to do either of these:

```ts
import kleur = require('kleur');
// or
import * as kleur from 'kleur';
// or, pick colors
import { green, bold } from 'kleur';
```

Reply:
I've worked around it in my project by doing this in my definition file

```ts
import kleur from 'kleur'

export interface ITextStyle {
    indention: number
    padding: boolean
    styles: typeof kleur.bgBlack[]
}
```

but i think i have got a idea on how to solve it, by defining a `declare module "kleur/types" {}` which can be imported in ts environments.

With the example of above it would be written like this

```ts
import { Color, Kleur } from 'kleur/types'

export interface ITextStyle {
    indention: number
    padding: boolean
    styles: Color[]
}
```

Reply:
I think this should solve the issue i tested it with the following scenarios

- `const kleur = require('kleur')`
- `import kleur from 'kleur'`
- `import { Color, Kleur } from 'kleur/types'`
- and like this

```ts
import kleur from 'kleur'
import { Color, Kleur } from 'kleur/types'

function test (kleurInput: Kleur): Color {
  return kleur.bgBlack
}
```